src/__init__.py

```diff
@@ -87,6 +87,12 @@ def root(netid):
 	return render_template("home.html", netid=netid)
 
 
+@app.context_processor
+@auth.get_auth_user
+def inject_header_values(netid):
+	is_staff = netid and common.is_staff(netid)
+	return dict(show_staff=is_staff)
```

Comment:
I think it still makes sense to pass the variable in as `is_staff` instead of `show_staff`.

Reply:
Noted!